src/libsecutils/include/secutils/credentials/verify.h

```diff
@@ -71,6 +72,7 @@ int CREDENTIALS_print_cert_verify_cb(int ok, X509_STORE_CTX* ctx_x509);
 /* TODO DvO remove this function when the ones using it are merged upstream */
 bool verify_cb_cert(X509_STORE_CTX* store_ctx, X509* cert, int err);
 
+#ifdef SECUTILS_USE_UTA
```

Comment:
No, this function needs to be available also without the UTA API !
Note that the `ura_ctx` is unused as documented. It is there just for backward compatibility.